SDPi_Supplement/asciidoc/volume2/dev-25/tf2-ch-a-mdpws-dev-25.adoc

```diff
@@ -0,0 +1,46 @@
+:var_uri_ws_discovery_transfer: https://www.w3.org/Submission/2006/SUBM-WS-Transfer-20060927/#Get
+:var_uri_dpws: http://docs.oasis-open.org/ws-dd/dpws/wsdd-dpws-1.1-spec.html
+[#vol2-clause-appendix-mdpws-dev-25]
+==== MDPWS: Discover BICEPS Services [DEV-25]
+
+===== GetMetadata Message
+
+The GetMetadata message is encoded by using {var_uri_ws_discovery_transfer}[WS-Get].
+
+====== Referenced Standards
+
+* <<ws-transfer-2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]
+* <<dpws-2009>> {var_uri_dpws}[Section 4]
+
+====== Message Outline
+.GetMetadata message
+[source#vol2_clause_appendix_a_mdpws_dev_25_transfer_get,xml]
+----
+include::../../listings/vol2-clause-appendix-a-mdpws-dev-25-transfer-get.xml[]
+----
+
+====== Message Semantics
+The GetMetadata message does not contain any further semantics.
+
+===== GetMetadataResponse Message
+
+The GetMetadataResponse message is encoded by using {var_uri_ws_discovery_transfer}[WS-Get].
+
+====== Referenced Standards
+
+* <<ws-transfer-2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]
+* <<dpws-2009>> {var_uri_dpws}[Section 4]
+
+====== Message Outline
+.GetMetadataResponse message

```

Comment:
This section seems to be undone.

1. The Relationship metadata section is missing from the message
2. The message semantics section misses the relatationship information as well
3. The message semantics reads odd, e.g. one item is rendered as 
"The SOMDS Provider's maps to the [message_semantics_discover_biceps_services_model] metadata."